packages/web-pkg/src/composables/actions/files/useFileActionsCopyPermanentLink.ts

```diff
@@ -34,7 +37,7 @@ export const useFileActionsCopyPermanentLink = () => {
         return copyLinkToClipboard(permalink)
       },
       isVisible: ({ space, resources }) => {
-        if (isPublicSpaceResource(space)) {
+        if (isPublicSpaceResource(space) || isLocationTrashActive(router, 'files-trash-generic')) {
```

Comment:
Could you check for `isTrashResource(resources[0])` instead? This feels more robust than these route checks.

Reply:
👍